corehq/apps/case_importer/views.py

```diff
@@ -332,5 +342,13 @@ def _bulk_case_upload_api(request, domain):
             search_field=search_field,
             create_new_cases=create_new_cases)
 
-    case_upload.trigger_upload(domain, config)
-    return json_response({"msg": "success"})
+    case_upload.trigger_upload(domain, config, comment=upload_comment)
+
+    upload_id = case_upload.upload_id
+
+    status_url = "{}{}".format(
+        get_url_base(),
+        reverse('case_importer_upload_status', args=(domain, upload_id))
+    )
```

Comment:
nit:

```suggestion
    status_url = absolute_reverse('case_importer_upload_status', args=(domain, upload_id))
```
Will require importing `absolute_reverse`

Reply:
Updated. For my knowledge: Is this just DRY, or does this do something different? Pulled this from somewhere else, so good for me to know if I should always use the `absolute_reverse` pattern or whether a non-absolute reverse is sometimes the right thing

Reply:
It is just DRY. `absolute_reverse` should only be used when the URL is being sent in an external response, otherwise `reverse` should be used.